src/components/SecondaryPanes/Scopes.js

```diff
@@ -109,7 +109,7 @@ export default connect(
     const { scope: frameScopes, pending } = getFrameScope(
       state,
       selectedSource && selectedSource.get("id"),
-      selectedFrame.id
+      selectedFrame ? selectedFrame.id : null
```

Comment:
we should have an invariant higher up

e.g. I'm surprised this passed. We can change the definition of isPaused i suppose to
https://github.com/devtools-html/debugger.html/blob/master/src/components/SecondaryPanes/index.js#L240-L247